File: front-end/src/model/api/hotel/dashboard/ReservationBase.ts
export class ReservationBase {
    public Id: string | undefined
    public RoomId: string | number | undefined
    public RoomNumber: number | undefined
    public RoomName: string | undefined
    public CheckIn: Date | undefined
    public CheckOut: Date | undefined
    public HolderId: string | number | undefined
    public HolderName: string | undefined
    public HolderSurname: string | undefined
    public Adults: number = 0
    public Childrens: number = 0
    public Babies: number = 0
}

export default ReservationBase

File: back-end/One.iGS.API.Models/Hotel/Dashboard/ReservationBase.cs
using System;

namespace One.iGS.API.Models.Hotel.Dashboard
{
    public class ReservationBase
    {
        public string Id { get; set; }
        public short RoomNumber { get; set; }
        public string RoomName { get; set; }
        public DateTime CheckIn { get; set; }
        public DateTime CheckOut { get; set; }
        public string HolderId { get; set; }
        public string HolderName { get; set; }
        public string HolderSurname { get; set; }
        public short Adults { get; set; }
        public short Childrens { get; set; }
        public short Babies { get; set; }
    }
}
